munit/shared/src/main/scala/munit/MUnitRunner.scala

```diff
@@ -200,6 +201,16 @@ class MUnitRunner(val cls: Class[_ <: Suite], newInstance: () => Suite)
     if (!filter.shouldRun(description)) {
       return Future.successful(false)
     }
+    if (suiteAborted) {
+      notifier.fireTestAssumptionFailed(
+        new Failure(
+          description,
+          new IllegalStateException("Suite has been aborted.")
```

Comment:
Can we use `FailSuiteException`?

Reply:
Let's remove the dot
```suggestion
          new IllegalStateException("Suite has been aborted")
```

Reply:
Done!